DEVELOPERS.md

```diff
@@ -276,3 +298,39 @@ An intermediate step generates a JSON file (`public_docs.json`) containing the d
 To generate this file, run:
 
     just generate-docs
+
+### Updating the main OpenSAFELY documentation repository
+
+:warning: Currently, changes made to the documentation in Data Builder's repository will bedeployed whenever any pull request in the main [documentation](https://github.com/opensafely/documentation) repository is next merged.
```

Comment:
missing a space between "be" and "deployed"

Reply:
Fixed; thanks!